examples/planout-example/Readme.md

```diff
@@ -78,6 +78,7 @@ To run this example go to it's root directory and run `npm install`.
 
 You can then run:
 
+    $ npm run build
```

Comment:
I don't think this is correct. This is running v1 version. Since this planout-example is covering only v1, I suppose it is correct but you should update the README of full-example to limit the scope.